shell.nix

```diff
@@ -36,24 +34,13 @@ pkgs.mkShell {
       kubernetes-helm
       yq-go # for manipulating golangci-lint config
     ]
-    ++ [
```

Comment:
@patricios-space @jadepark-dev fyi, these dependencies are moved form the default shell to specific shells:

- contract shell: 
    - Jetton contacts as a dependency
- ccip-e2e shell: 
    - TODO: missing CCIP contracts as a dependency (currently implicit via CI build run)
    - Jetton contacts as a dependency
    - chainlink-ton as a dependency